lib/chef/resource/powershell_script.rb

```diff
@@ -54,6 +69,27 @@ def convert_boolean_return(arg = nil)
       def self.get_default_attributes(opts)
         { convert_boolean_return: true }
       end
+
+      # Options that will be passed to Windows PowerShell command
+      def default_flags
+        return "" if Chef::Platform.windows_nano_server?
```

Comment:
We should decide later if we want to still support all these windows nano checks